src/eckit/stats/Distribution.h

```diff
@@ -0,0 +1,85 @@
+/*
+ * (C) Copyright 1996- ECMWF.
+ *
+ * This software is licensed under the terms of the Apache Licence Version 2.0
+ * which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * In applying this licence, ECMWF does not waive the privileges and immunities
+ * granted to it by virtue of its status as an intergovernmental organisation nor
+ * does it submit to any jurisdiction.
+ */
+
+
+#pragma once
+
+#include <cstddef>
+#include <iosfwd>
+#include <string>
+
+
+namespace eckit {
+class Parametrisation;
+}
+
+
+namespace eckit::stats {
+
+
+class Distribution {
```

Comment:
You're right -- this is a class to generate noise onto a field according to specific distributions. It might not have a direct use here, but it is important functionality to support statistics testing (incl. downstream packages -- I use this in MIR to generate training data, although this isn't the greatest use). I used this to test the statistics when developing, and I haven't ported those tests as they should be done anew (they aren't useful as is)...